docs/openapi3.0.yml

```diff
@@ -319,11 +340,17 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/CreateQuestion'
+      responses:
+        '200': 
+          description: 'Question successfully created'
```

Comment:
also return the id of the question

Reply:
@MichalskyL 
Should it just the id, or the whole question Object?
Alternative 1:
```
{
  "userId": "string"
}
```
Alternative 2:
```
{
  "id": "string",
  "content": "string",
  "likes": 0,
  "dislikes": 0,
  "created": "string",
  "opinion": "like",
  "title": "string",
  "tags": [
    "string"
  ],
  "numberOfAnswers": 0,
  "updated": "string",
  "author": {
    "id": "string",
    "name": "string",
    "type": "guest"
  }
}
```

Reply:
the whole object